frontend/src/scenes/billing/ExportsUnsubscribeTable/exportsUnsubscribeTableLogic.tsx

```diff
@@ -100,7 +100,7 @@ export const exportsUnsubscribeTableLogic = kea<exportsUnsubscribeTableLogicType
                         description: pluginConfig.description,
                         icon: <RenderApp plugin={pluginConfig.plugin_info} imageSize="small" />,
                         disabled: !pluginConfig.enabled,
-                        url: urls.projectApp(pluginConfig.plugin),
+                        url: urls.pipelineNode(PipelineStage.SiteApp, pluginConfig.id),
```

Comment:
I was a little unsure about this and to a lesser degree the `urls.pipelineNode(PipelineStage.Destination, batchExportConfig.id)` change.

It seems in the past you could do `/apps/:id` as a generic route to any app, but now you have to know the type in advance. 

I am assuming that batch exports always live inside the `Destination`. I was less sure if `pluginConfigsToDisable` would always be "site apps" but read that they were no transformation / filter plugins:
https://github.com/PostHog/posthog/blob/a422e7ea4b0feed8ab1d40eb240d1c59899bacf0/posthog/api/plugin.py#L390-L391

And if batch exports are handled separately I assumed they would be `SiteApp`s

Reply:
Yeah they are all in destination really as it is destinations that are charged in this way now. It might have issues but whatever we can fix forward